flipkart/index.html

```diff
@@ -0,0 +1,711 @@
+            <br>
+            <input type="checkbox" id="rating" name="rating" value="rating">
+            <label for="rating"> 2* & above</label>
+            <br>
+            <input type="checkbox" id="rating" name="rating" value="rating">
+            <label for="rating"> 1* & above</label>
+          </form>
+        </div>
+        <hr>
+        <div class="internal-storage">
+          <h4>INTERNAL STORAGE</h4>
+          <form>
+            <input type="checkbox" id="storage" name="storage" value="storage">
+            <label for="storage">256 GB and above</label>
+            <br>
+            <input type="checkbox" id="storage" name="storage" value="storage">
+            <label for="storage">128-255.9GB</label>
+            <br>
+            <input type="checkbox" id="storage" name="storage" value="storage">
+            <label for="storage">64-127.9GB</label>
+            <br>
+            <input type="checkbox" id="storage" name="storage" value="storage">
+            <label for="storage">32-63.9GB</label>
+            <br>
+            <input type="checkbox" id="storage" name="storage" value="storage">
+            <label for="storage">16-31.9GB</label>
+            <br>
+          </form>
+        </div>
+        <hr>
+        <div class="battery">
+          <h4>BATTERY CAPACITY</h4>
+        </div>
+        <hr>
+        <div>
+          <h4>OPERATING SYSTEM</h4>
+        </div>
+        <hr>
+        <div>
+          <h4>NETWORK TYPE</h4>
+        </div>
+        <hr>
+        <div>
+          <h4>SCREEN SIZE</h4>
+        </div>
+        <hr>
+        <div>
+          <h4>PRIMARY CAMERA</h4>
+        </div>
+        <hr>
+        <div>
+          <h4>SECONDARY CAMERA</h4>
+        </div>
+        <hr>
+        <div>
+          <h4>SIM TYPE</h4>
+        </div>
+        <hr>
+        <div>
+          <h4>CLOCK SPEED</h4>
+        </div>
+        <hr>
+        <div>
+          <h4>PROCESSOR BRAND</h4>
+        </div>
+        <hr>
+        <div>
+          <h4>NUMBER OF CORES</h4>
+        </div>
+        <hr>
+        <div>
+          <h4>SPECIALITY</h4>
+        </div>
+        <hr>
+        <div>
+          <h4>AVAILABILITY </h4>
+        </div>
+        <hr>
+        <div class="discount">
+          <h4>OFFERS</h4>
+          <form>
+            <input type="checkbox" id="offers" name="offers" value="offers">
+            <label for="offers">Bank Offer</label>
+            <br>
+            <input type="checkbox" id="offers" name="offers" value="offers">
+            <label for="offers">Exchange Offer</label>
+            <br>
+            <input type="checkbox" id="offers" name="offers" value="offers">
+            <label for="offers">No cost EMI</label>
+            <br>
+            <input type="checkbox" id="offers" name="offers" value="offers">
+            <label for="offers">Special price</label>
+          </form>
+        </div>
+      </div>
+    </div>
+    <div class="section2">
```

Comment:
This is not what we meant by sections, Section should be separated by <section> Tag.